apps/AppWithWearable/pubspec.yaml

```diff
@@ -101,7 +101,7 @@ dev_dependencies:
   envied_generator: ^0.5.4+1
   build_runner: ^2.4.11
   objectbox_generator: any
-
+  flutter_flavorizr: ^2.2.3
```

Comment:
I will prefer a vanilla approach here rather than going for a package. It is nice to use libraries but IMO, it shouldn't bring any blocker during the release. Here I can see few issues in this library:

https://github.com/AngeloAvv/flutter_flavorizr/issues/264
https://github.com/AngeloAvv/flutter_flavorizr/issues/161

Additionally, it doesn't have that much active support.

Reply:
Yes, it shoudlnt pose any threat, it can equally be removed after being used.